foodnet/membership/fixtures/initial_data.json

```diff
@@ -0,0 +1,27 @@
+[
+{
+    "model": "membership.membercategory",
+    "pk": 1,
+    "fields": {
+        "name": "houshold"
+    }
+},
+{
+    "model": "membership.membercategory",
+    "pk": 2,
+    "fields": {
+        "name": "school"
+    }
+},
+{
+    "model": "membership.division",
+    "pk": 1,
+    "fields": {
+        "allow_webmembers": true,
+        "category": "food",
+        "name": "default",
+        "contact": "admin@localhost",
+        "shortname": "def"
+    }
+}
+]
```

Comment:
I would do initial data as a data migration. https://docs.djangoproject.com/en/1.8/topics/migrations/#data-migrations
Since 1.7, Django doesn't load fixtures automatically - but you probably know that. I guess this is for development purposes?